packages/labs/analyzer/rollup.config.typescript.js

```diff
@@ -0,0 +1,25 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Why is this extraneous? It's called on line 20

Reply:
eslint was yelling that `@rollup/plugin-commonjs` should be in `dependencies` instead of `devDependencies`. I didn't take the time to see if a config change would help.

Reply:
there's a way to escape that here. (the labs/react rule should be removed as it no longer applies)
https://github.com/lit/lit/blob/cabe72a863a5de14a4bbca384374db748dd9b4c5/.eslintrc.json#L41-L47

Reply:
or here
https://github.com/lit/lit/blob/cabe72a863a5de14a4bbca384374db748dd9b4c5/.eslintrc.json#L82-L104

Reply:
I applied both of @augustjk's suggested changes